sim/pokemon.ts

```diff
@@ -19,6 +19,7 @@ interface MoveSlot {
 	disabledSource?: string;
 	used: boolean;
 	virtual?: boolean;
+	originalPoke?: number; // Partners in Crime
```

Comment:
im unsure how i feel about adding stuff used just for oms onto these values. @Marty-D thoughts?

Reply:
Not a huge fan, what makes this necessary now when it wasn't necessary to implement this OM back in October 2019?

Reply:
I believe the reason why PiC was removed was because a ton of changes to the sim were made, so I don't think a property like this was needed before. I wonder if using `pokemon.m` can be used for this though?

Reply:
> I wonder if using pokemon.m can be used for this though?
It's possible. Just pushed the an alternative solution to Transform/mimic support that doesn't interfere with the core sim files.